lib/tapjoy/autoscaling_bootstrap/AWS/Autoscaling/launch_config.rb

```diff
@@ -16,7 +16,7 @@ def delete
 
             def create(image_id:, instance_type:, security_groups:, user_data:,
               keypair:, iam_instance_profile:, classic_link_vpc_id: nil,
-              classic_link_sg_ids: nil, **unused_values)
+              classic_link_sg_ids: nil, spot_price: nil, ebs_optimized: true, **unused_values)
```

Comment:
The docs say the default for this option is `false`, but it looks like it's `true` here